Wikipedia/Code/WMFSearchResultsTableViewController.m

```diff
@@ -73,4 +73,9 @@ - (WMFEmptyViewType)emptyViewType {
     return WMFEmptyViewTypeNone; //Is controlled by the search VC
 }
 
+- (NSString*)analyticsContext {
+    return @"Search";
```

Comment:
what's the difference between this and the `analyticsContext` provided by `WMFSearchViewController`? if they're actually meant to indicate the same context, they should be using a global constant.